drivers/lte_link_control/lte_lc.c

```diff
@@ -423,6 +421,116 @@ int lte_lc_edrx_req(bool enable)
 	return 0;
 }
 
+/**@brief Parses an AT command response, and returns the current network
+ *	  registration status if it's available in the string.
+ *
+ * @param at_response Pointer to buffer with AT response.
+ * @param status Pointer to where the registration status is stored.
+ *
+ * @return Zero on success or (negative) error code otherwise.
+ */
+static int parse_nw_reg_status(char *at_response,
+			       enum lte_lc_nw_reg_status *status)
+{
+	int err, reg_status;
+	struct at_param_list resp_list = {0};
+	char  response_prefix[sizeof(AT_CEREG_RESPONSE_PREFIX)] = {0};
+	size_t response_prefix_len = ARRAY_SIZE(response_prefix);
```

Comment:
Won't `sizeof` be more natural for this case than `ARRAY_SIZE`?

Reply:
Yep, fixed all of them. Thanks